scipy/stats/_continuous_distns.py

```diff
@@ -8648,13 +8655,24 @@ def _cdf(self, x, a, b):
         return np.exp(self._logcdf(x, a, b))
 
     def _logcdf(self, x, a, b):
-        return _log_gauss_mass(a, x) - _log_gauss_mass(a, b)
```

Comment:
This was another source of catastrophic cancellation - when the terms are approximately equal but not nearly zero. When this difference of the two terms is nearly zero (`>-0.1`, say), we avoid the catastropic cancelation by working through the survival function. We implement the same idea in `_logsf` (but avoid infinite back and forth calls between the two functions by choosing `-0.1 > log(1/2)`).